hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveSchemaUtil.java

```diff
@@ -154,6 +154,22 @@ public void testComplexTypeAndTypeInfoConvert() {
     }
   }
 
+  @Test
+  public void testConversionWithoutLastComment() {
+    Schema expected = new Schema(
+            optional(0, "customer_id", Types.LongType.get(), "customer comment"),
+            optional(1, "first_name", Types.StringType.get(), null)
```

Comment:
Can you please fix the formatting in this change? Continuation indents are 2 indents / 4 spaces.

Reply:
done